README.md

```diff
@@ -66,3 +66,87 @@ is written in `aggregate.csv`.
 ## Per-method Checking
 
 See examples in the `tests` directory.
+
+## Package History Checking
+
+There is a suite of scripts to examine the git history of packages 
+and report how the stability of methods changed over time. 
+These are located in `$JULIA-STS/scripts/timeline`:
+
+- `batch.jl`: The main script, this is where all the packages to check are listed.
+- `history.jl`: Takes care of git management (clones the package, checks out individual commits). Also the only part that's using multiple threads to speed up the processing.
+- `process-package.jl`: Runs the stability checks for a given package for a given revision (a slightly modified version of `scripts/process-package.jl`).
+- `aggregate.jl`: Creates a CSV summary of all the processed revisions of a package.
+- `filter.jl`: Tries to filter the interesting stability cases with a heuristic.
+
+These scripts assume the following packages: `CSV` and `DataFrames` (for filtering), `TOML` (for history to parse `Project.toml`s), `Plots` and `UnicodePlots` (for history to report balancing stats, can be deleted)
+
+For example, to check the `Multisets` package using 8 threads, first modify the `batch.jl` 
```

Comment:
The `modify` bit is a little unpleasant, especially if we are to do it at scale. It's better take a parameter (or env var) with a file name with the list of packages to process.

Also, there's no reason to specify the github repo url: Julia has this info, even if in a bit unfriendly form. See if you can wrap this in a function:

```
julia> using Pkg

julia> general=Pkg.Registry.reachable_registries()[1]       # NOT PRETTY but normally should Just Work™
Registry: "General" at "/home/artem/.julia/registries/General.toml":
  uuid: 23338594-aafe-5451-b93e-139f81909106
  repo: https://github.com/JuliaRegistries/General.git
  git-tree-sha1: 998e28e3bb762887f738913272e235131e93de60
  packages: 9170

julia> uuid=findfirst(p -> p.name == "Multisets", general.pkgs)
UUID("3b2b4ff1-bcff-5658-a3ee-dbcf1ce5ac09")

julia> pkg_info=Pkg.Registry.registry_info(general[uuid])
...
julia> pkg_info.repo
"https://github.com/scheinerman/Multisets.jl.git"
```